examples/01-basic-static-list/README.md

```
@@ -0,0 +1,165 @@
+```bash
+kubectl apply -f listsource.yaml
+```
+
+Check that the ConfigMap was created:
+
+```bash
+kubectl get configmap basic-list-data -o yaml
+```
+
+You should see a ConfigMap containing:
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)